Examples/Python/python/acts/examples/itk.py

```diff
@@ -390,32 +393,35 @@ def itkSeedingAlgConfig(
         interactionPointCut = True
         impactMax = 2 * u.mm
         zBinsCustomLooping = [
-            1,
             2,
             3,
             4,
+            5,
+            12,
```

Comment:
why do you change `zBinsCustomLooping`?

Reply:
Just increased by 1 all the numbers due to the addition of a bin in the leftmost part of the detector

Reply:
ah! off course!